README.md

```diff
@@ -121,3 +121,43 @@ $ ssh USERNAME@USERNAME@IP-ADDRESS ## this is the managed node IP address. Once
 10. Create a file called "todoapp.service on the control node and include the details as per todoapp.service
 
 11. Start the app by running ansible-playbook YOUR_PLAYBOOK_FILE -i YOUR_INVENTORY_FILE. then visit the site in your browser using the IP address of the host VM followed by ":5000"
+
+## Docker
+
+### Build Development and Production containers
+
+To build development images and containers, do the following:
+
+```
+$ docker-compose up development
+```
+
+Above builds the "development" stage from the Dockerfile, and docker-compose.yml tags the image as "todo-app:dev".
+
+The local:container ports will be mapped to 5000:5000 by default
+
+To build production images and containers, do the following:
+
+```
+$ docker-compose up production
+```
+Above builds the "production" stage from the Dockerfile, and docker-compose.yml tags the image as "todo-app:prod".
+
+The local:container ports will be mapped to 5001:5000 by default
```

Comment:
This is the other way around in your Docker compose file (dev -> 5001, prod -> 5000)